tracex/extraction/views.py

```diff
@@ -47,14 +44,52 @@ def form_valid(self, form):
 
         return response
 
-class ProcessingView(generic.TemplateView):
-    """View for processing the patient journey."""
 
-    template_name = "processing.html"
+class JourneyFilterView(generic.FormView):
+    """View for selecting extraction results filter"""
 
-    def get(self, request, *args, **kwargs):
-        """Redirect to result page."""
-        return redirect("result")
+    form_class = FilterForm
+    template_name = "filter_journey.html"
+    success_url = reverse_lazy("result")
+
+    def form_valid(self, form):
+        """Save the uploaded journey in the cache."""
```

Comment:
I'm not realy confident with django views but the docstring from this method suggests to me that when i validate the form the journey will be saved to the cache and something else happens like that i am redirected to the next view. However in the code below you can see that the run method from the orchestrator is called hence our pipeline is started. Can you make the code more clear here?